toqito/matrices/tests/test_cyclic_permutation.py

```diff
@@ -0,0 +1,26 @@
+"""Test cyclic_permutation."""
+import numpy as np
+
+from toqito.matrices import cyclic_permutation_matrix
+
+
+def test_cyclic_permutation():
+    """Test cyclic permuation matrix."""
+    n = 10
+    res = cyclic_permutation_matrix(n)
+    bool_mat = np.allclose(np.linalg.matrix_power(res, n), np.eye(n))
+    np.testing.assert_equal(np.all(bool_mat), True)
```

Comment:
You can get rid of this. Personally, I think this type of check is redundant. 

You can check expected matrix elements are close to the calculated matrix elements and leave it at that.

Reply:
Fair point.